packages/components/pie-checkbox/src/index.ts

```diff
@@ -153,12 +164,13 @@ export class PieCheckbox extends FormControlMixin(RtlMixin(LitElement)) implemen
                 .indeterminate=${!!indeterminate}
                 aria-label=${aria?.label || nothing}
                 aria-labelledby=${label ? nothing : aria?.labelledby || nothing}
-                aria-describedby= ${aria?.describedby || nothing}
+                aria-describedby=${ifDefined(assistiveText ? assistiveTextIdValue : undefined)}
                 @change=${this.handleChange}
                 data-test-id="checkbox-input"
             />
             ${label}
-        </label>`;
+        </label>
+        ${assistiveText ? html`<pie-assistive-text id="${assistiveTextIdValue}" variant=${ifDefined(status)} data-test-id="pie-checkbox-assistive-text">${assistiveText}</pie-assistive-text>` : nothing}`;
```

Comment:
I'm not seeing the status get passed down to assistive text on the Storybook preview btw

Reply:
![Uploading Screenshot 2024-06-19 at 08.46.55.png…]()

Reply:
<img width="1487" alt="Screenshot 2024-06-19 at 10 59 44" src="https://github.com/justeattakeaway/pie/assets/168754213/a9d0e4f5-aeba-45a6-90e7-72be5460ff66">

https://pr1516-storybook.pie.design/?path=/story/checkbox--default&args=status:success;assistiveText:Test

Reply:
Strange, when I follow the exact link you posted:
<img width="961" alt="Screenshot 2024-06-19 at 09 02 03" src="https://github.com/justeattakeaway/pie/assets/16766185/32555c6b-323b-4bd8-b2e3-e1ad97c52aad">

Reply:
Adding the recording here for others to see. I think the issue is the assistive text is not imported into the checkbox story, so the styles and icons don't appear. Hovering the assistive text story pre-loads the assets before we click it, which makes it appear properly in the checkbox story on hover.

I think the fix is importing assistive text into the checkbox file.

https://github.com/justeattakeaway/pie/assets/16766185/74158af3-b956-45cf-9f02-0697fdd3c956